apps/web/components/SimpleSlider/SimpleSlider.tsx

```diff
@@ -26,6 +32,8 @@ interface SimpleSliderProps {
   breakpoints?: SlideState['breakpoints']
   slideWidthOffset?: SlideState['slideWidthOffset']
   items: ReactNode[]
+  carousleContoler?: boolean
```

Comment:
`carouselController`?

Reply:
Yes, its a couple of red dots that allow you to jump between slides, it was only visible above the slider below small, but in the design for this page it had to be below it and always visible. So I added it a tiny customisation to the simple slider to be able to do this.

Reply:
I was suggesting to fix the spelling of the word 😛